spec/features/user_creates_a_published_strategy_spec.rb

```diff
@@ -49,11 +49,11 @@
       within('#strategy_viewers_accordion') do
         find('.accordion').click
         find('.tagAutocomplete').set('Ally 0')
-        page.find('.tagAutocomplete').native.send_keys(:return)
```

Comment:
Similar question as above.

Reply:
This relates to the issue I'm having with the form submit within the 'viewers_accordion'